lib/config/locales/en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  # For ActiveRecord parity.
```

Comment:
The message should be here because this is where it's needed for Mongoid, not because this is where AR keeps it.